test/end-to-end.js

```diff
@@ -99,11 +95,11 @@ module.exports = function (opts) {
 
           // bob can also decrypt
           var content = ssbKeys.unbox(ctxt, alice.private)
-          t.deepEqual(content, {
-            type: 'secret2',
-            secret: "it's a secret!",
-            recps: [alice.id]
-          }, 'alice can decrypt')
+          t.deepEqual(
+            content,
+            { type: 'secret2', secret: "it's a secret!", recps: [alice.id] },
```

Comment:
:fire: this test does something very strange (I think).
if you compare what was published with `feed.add` and what comes out the the private box ... they're different!

The RECPS field has been array'd !
This is fine, so long as it doesn't mess with signature / hashes of content?
Maybe I don't understand the order of that relative to encryption though

Reply:
_I didn't change the test, just the layout to make it easy to compare. The code I wrote still passes this strange test_